index.html

```diff
@@ -17,7 +27,7 @@ <h2>IESEG Location</h2>
         <a href="#Sign-Up">Sign-Up</a>
       </div>
     </div>
-    <a href="#profile">Profil</a>
+    <a href="#profil">Profil</a>

```

Comment:
why putting profil without an e ? I think the proper way of spelling it with an e

Reply:
Indeed, I'm changing it !